### DISCUSSION TITLE
아이템 적용 방법에 대해 질문드립니다.

### DISCUSSION THREAD

**Discussion by leehearin** [naverz/zepeto-studio-kor#69] [5. Q&A]
NPC를 클릭하면  아이템(탈것)이 나타나서  캐릭터에 적용시키고 싶은데요. 

예를 들어 자동차라고 하면

1 NPC 클릭
2 자동차 나타남
3 캐릭터가 자동차를 탄다
4 캐릭터가 자동차를 타고 맵 내에서 움직일 수 있다.

이에 대해 적용할 수 있는 방법이 있을까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요

말씀해 주신 부분들은 기능 지원 예정이 없기 때문에 직접 구현해서 진행 부탁드립니다.
